Head line: add cli implement first version.

Yihen-Liu opened the pull request:
add cmd package for cli module.
  restruct main.go file.

  Signed-off-by:liuyihen@onchain.com

Yihen-Liu committed:
Head line: add cli implement first version.
  add cmd package for cli module.
  restruct main.go file.

  Signed-off-by:liuyihen@onchain.com

laizy commented on account/client.go:
```diff
@@ -426,11 +434,18 @@ func GetClient() Client {
+		client = Open(WALLET_FILENAME, passwd)
+	}
+
+	if client == nil {
```
meaningless condition check , return client directly.

laizy commented on cmd/info_cmd.go:
```diff
@@ -0,0 +1,221 @@
+
+func blockInfoCommand(ctx *cli.Context) error {
+	/*init ledger actor for info command*/
+	ledger.DefLedger, _ = ledger.NewLedger()
```
info should query from restful api , can not get from db directly

laizy replied:
because command is executed in a separate process

Yihen-Liu replied:
Thx.

Yihen-Liu committed:
<style>: Delete and correct invalid statement
  1. delete unnecessary print in main.go
  2. delete invalid return statement in NewClient()
  3. correct sub-command describtion

  Signed-off-by: liuyihen@onchain.com

laizy commented on cmd/utils/flags.go:
```diff
@@ -0,0 +1,351 @@
+
+	RinkebyFlag = cli.BoolFlag{
+		Name:  "rinkeby",
+		Usage: "Rinkeby network: pre-configured proof-of-authority test network",
```
what's this?

Yihen-Liu commented:
New version has been given pull request.
Close this one.
